src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -541,20 +541,26 @@ def get(self, bucket, daemon_name=None):
         result = self.proxy(daemon_name, 'GET', 'bucket', {'bucket': bucket})
         bucket_name = RgwBucket.get_s3_bucket_name(result['bucket'],
                                                    result['tenant'])
+        owner = result['owner']
+        try:
+            instance = RgwClient.admin_instance(daemon_name=daemon_name)
+            instance.proxy('GET', 'user', {'uid': result['owner']}, None)
```

Comment:
Is this only to verify if the owner is a user not owned by an account?
_SUGESTION:_ 
I think we could try to achive that by getting the list of rgw accounts, which we could keep in memory, and check against it that the user is not in that list, otherwise assign the `dashboard` user

Reply:
Yeah.. makes sense, thanks for the update!